java-squid/src/main/java/org/sonar/java/se/SymbolicValue.java

```diff
@@ -386,4 +401,24 @@ public String toString() {
       return leftOp + " ^ " + rightOp;
     }
   }
+
+  public static class ResourceWrapperSymbolicValue extends SymbolicValue {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Override this superclass' "equals" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2160)